src/main/scala/org/zalando/spearheads/innkeeper/routes/DeleteRoute.scala

```diff
@@ -38,15 +35,15 @@ class DeleteRoute @Inject() (
       logger.info(s"try to $reqDesc")
 
       hasOneOfTheScopes(authenticatedUser, reqDesc, scopes.WRITE) {
-        findRoute(id, routesService, "delete /routes/{}")(executionContext) { route =>
-          logger.debug(s"try to delete /routes/$id route found $route")
+        findRouteOwnerTeam(id, routesService, "delete /routes/{}")(executionContext) { routeTeam =>
```

Comment:
It's because we only have the route id here as opposed to PostRoutes where you have the pathId in your route.